src/tensorlake/functions_sdk/functions.py

```diff
@@ -212,21 +155,17 @@ def construct(fn):
 class FunctionCallResult(BaseModel):
     ser_outputs: List[TensorlakeData]
     traceback_msg: Optional[str] = None
-
-
-class RouterCallResult(BaseModel):
-    edges: List[str]
-    traceback_msg: Optional[str] = None
+    metrics: Optional[Metrics] = None
+    output_encoding: str
+    next_nodes: Optional[List[str]] = None
```

Comment:
Should the next node only be a single node instead of a list?
If the next_node is None and there are more the two edges coming out of the current node then this is a runtime error right? Or we take the first edge by default?